legal-api/src/legal_api/services/authz.py

```diff
@@ -168,8 +168,7 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
             'correction': {
                 'legalTypes': ['CP', 'BEN', 'SP', 'GP', 'BC', 'ULC', 'CC'],
                 'blockerChecks': {
-                    'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
-                    'business': [BusinessBlocker.DEFAULT]
```

Comment:
I don't think we should be removing `'business': [BusinessBlocker.DEFAULT]`.  

What is the motivation for removing the the default blockers(draft/pending filing + admin freeze) here?

The current allowable actions logic for corrections was previously reviewed via the [allowable actions spreadsheet](https://docs.google.com/spreadsheets/d/1sxsZ7vuODtD_ftjR27qLvgYErWDB-msvzGucjvBRQXQ/edit#gid=0).

![image](https://github.com/bcgov/lear/assets/6685223/d8ce9c5b-7be2-4845-9e0e-39257c304f36)

Reply:
Thanks Argus! based on this doc, yes I think I can't remove it. I'll close this PR.